src/options/optionsMain.ts

```diff
@@ -139,7 +141,7 @@ export async function getOptions(rawOptions: RawOptions): Promise<AppOptions> {
       'is recommended only for troubleshooting or if you like Shakespeare.',
     );
   } else {
-    log.setLevel('info');
+    log.setLevel(options.nativefier.quiet ? 'silent' : 'info');
```

Comment:
We're already in an if/else; for clarity please use an `else if` rather than a ternary